### PR TITLE
Fixes for optimism and otterscan bugs

### PR DESCRIPTION
**Pull request by pcw109550**
Optimism specific tweak
- Disable EIP1559 page
- Add L1 Gas Price, L1 Fee Scalar, L1 Gas Used By Txn for transaction page
- Label depositTx when `type = 0x7e`
- Handle error when RPC response does not include `tx.GasPrice`
- Error handle reward field on transaction page for depositTx

Otterscan || Erigon bug
- Fix post merge block reward logic: Fixes https://github.com/wmitsuda/otterscan/issues/397

This PR is only for code review. WILL NOT MERGE.

### CODE REVIEW

File: src/transaction/RewardSplit.tsx [hunk 1]
```diff
@@ -26,8 +26,9 @@ const RewardSplit: React.FC<RewardSplitProps> = ({ txData }) => {
     : BigNumber.from(0);
 
   const minerReward = paidFees.sub(burntFees);
-  const burntPerc =
-    Math.round(burntFees.mul(10000).div(paidFees).toNumber()) / 100;
+
+  // for depositTx, paidFees == 0 so ignore
+  const burntPerc = paidFees != BigNumber.from(0) ? 0 : Math.round(burntFees.mul(10000).div(paidFees).toNumber()) / 100;
```
Comment (ImTei) [inline src/transaction/RewardSplit.tsx]: This line should not work as respected.

**Review by ImTei — CHANGES_REQUESTED** [review]
It would be better if you makes l1 fee calculation as util function :)

File: src/transaction/RewardSplit.tsx (hunk 1 above)
Comment (pcw109550) [reply src/transaction/RewardSplit.tsx]: Fixed via https://github.com/testinprod-io/otterscan/pull/5

**Comment by pcw109550** [comment]
Closing because PR review done

**Pull request closed without merging**